app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -69,14 +69,16 @@ It’s important to note that, even if you decide to apply for QTS, you will hav
 
 ### How to apply for QTS
 
-From 1 February 2023, if you have a teaching qualification from the following countries and regions, you will be able to access a new service, called [Apply for qualified teacher status in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start), to check if you meet the requirements for QTS:
+If you have a teaching qualification from the following countries and regions, you can now access a new service, called [Apply for qualified teacher status in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start), to check if you meet the requirements for QTS and apply:
```

Comment:
Would it be better to have the list of countries before the cta?

```suggestion
If you have a teaching qualification from:

* Australia
* Canada
* the EEA
* Guernsey
* Gibraltar
* Hong Kong
* Jersey
* New Zealand
* Northern Ireland
* Scotland
* Switzerland
* Ukraine
* the USA

you can [check if you meet the requirements for QTS and apply](https://apply-for-qts-in-england.education.gov.uk/eligibility/start).
```

Reply:
Um, I'm not sure it is, because you don't know why you're scanning the list of countries for your country without the CTA at the top?

Reply:
But you could click on the link before you've even seen the list of countries, no?